l10n_it_fatturapa_in/models/account.py

```diff
@@ -48,18 +48,20 @@ class AccountInvoice(models.Model):
                  'date_invoice', 'type', 'efatt_rounding')
     def _compute_amount(self):
```

Comment:
Essendo il metodo originale decorato con `@api.one` non sarebbe più coerente usarlo anche nell'override?

Reply:
`@api.one` è deprecato, visto che altri moduli OCA ne tengono conto, [vedi](https://github.com/OCA/account-financial-tools/blob/12.0/account_invoice_currency/models/account_invoice.py#L20), mi adeguerei

Reply:
Non viene chiamato con più record però in questo caso? Altrimenti non avrebbe generato l'errore no?

Reply:
Metterei allora ` @api.multi` così è esplicito